src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colors color;
+    private int number;
+
+    public void setColor(Colors color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return this.color.toString();
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public int getNumber() {
+        return this.number;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString();
```

Comment:
The `toString()` method should be overridden to provide a meaningful string representation of the `Ball` object. Consider returning a string that includes both the color and number of the ball, for example: `return "Ball{color=" + color + ", number=" + number + "}";`.